portfolio/src/App.jsx

```diff
@@ -1,157 +1,145 @@
-              <div className="flex justify-evenly mt-12">
-                <button className="px-6 py-3 text-lg bg-gradient-to-t from-orange-500 rounded-full to-yellow-500 hover:from-orange-700 hover:to-yellow-700">
-                  Live preview
-                </button>
-                <button className="px-6 py-3 border rounded-full hover:border-orange-500 hover:text-yellow-500">
-                  Checkout github
-                </button>
-              </div>
-            </div>
-            <div className="border border-gray-500 rounded-md p-5">
-              {/*https://pwa-challenge1-b508091ee337.herokuapp.com/*/}
-              <a href="https://pwa-challenge1-b508091ee337.herokuapp.com/" target="_blank" class="project 4"></a>
-              <img src={Project4} alt="https://pwa-challenge1-b508091ee337.herokuapp.com/" />
-              <h3 className="text-2xl font-semibold mt-8">
-              This application features a number of data persistence techniques, insuring the application runs regardless of browser supported function. 
-              </h3>
-              <p className="text-gray-400 text-sm mt-4">https://github.com/tfkjosh/PWA-Challenge</p>
-              <div className="flex justify-evenly mt-12">
-                <button className="px-6 py-3 text-lg bg-gradient-to-t from-orange-500 rounded-full to-yellow-500 hover:from-orange-700 hover:to-yellow-700">
-                  Live preview
-                </button>
-                <button className="px-6 py-3 border rounded-full hover:border-orange-500 hover:text-yellow-500">
-                  Checkout github
-                </button>
-                </div>
-              </div>
+            <div className="flex flex-col sm:flex-col gap-12 mt-11">
+              {projects.map(
+                ({ imgSrc, description, githubHref, livePreviewHref }) => (
+                  <div className="border border-gray-500 rounded-md">
+                    <a href={githubHref} target="_blank">
+                      <img
+                        src={imgSrc}
+                        alt={githubHref}
+                        className="w-full h-auto"
+                      />
+                    </a>
+                    <div className="p-5">
+                      <h3 className="text-2xl font-semibold">{description}</h3>
+                      <div className="flex gap-4 justify-end mt-8">
+                        {livePreviewHref && (
+                          <a href={livePreviewHref} target="_blank">
+                            <button className="px-6 py-3 text-lg transition opacity-100 bg-gradient-to-t from-orange-500 rounded-full to-yellow-500 hover:opacity-90">
+                              Live preview
+                            </button>
+                          </a>
+                        )}
+
+                        <a href={githubHref} target="_blank">
+                          <button className="px-6 py-3 rounded-full opacity-100 border border-white">
```

Comment:
Updated the hover state of this button to be similar to that of the Live Preview button. Essentially, on hover, it just dims a bit instead of jumping to that orange color. Totally a preference thing, so if you prefer the orange border color with the yellow text on hover, more power to you. The only suggestion I would make in that case is to use the same color for both the border and the text. It makes the button look a bit more cohesively styled in the UI, in my opinion